buildSrc/src/main/groovy/com/uber/okbuck/core/dependency/DependencyCache.groovy

```diff
@@ -109,13 +110,21 @@ class DependencyCache {
             File sourcesJar = new File(dependency.depFile.parentFile, sourcesJarName)
 
             if (!sourcesJar.exists()) {
-                def sourceJars = rootProject.fileTree(
-                        dir: dependency.depFile.parentFile.parentFile.absolutePath,
-                        includes: ["**/${sourcesJarName}"]) as List
-                if (sourceJars.size() == 1) {
-                    sourcesJar = sourceJars[0]
-                } else if (sourceJars.size() > 1) {
-                    throw new IllegalStateException("Found multiple source jars: ${sourceJars} for ${dependency}")
+                if (FileUtils.directoryContains(rootProject.projectDir, dependency.depFile)) {
```

Comment:
Can you check if dependency is local first? There is a property `depenency.isLocal` and this logic should only apply to such dependencies